application/helpers/replacements_helper.php

```diff
@@ -683,6 +684,7 @@ function templatereplace($line, $replacements = array(), &$redata = array(), $de
     $coreReplacements['MOVENEXTBUTTON'] = isset($movenextbutton) ? $movenextbutton : '';    // global
     $coreReplacements['NOSURVEYID'] = (isset($surveylist))?$surveylist['nosid']:'';
     $coreReplacements['NUMBEROFQUESTIONS'] = $_totalquestionsAsked;
+    $coreReplacements['NUMBEROFGROUPS'] = $totalgroups;
```

Comment:
Where are such placeholders currently documented? I can't find NUMBEROFGROUPS at the manual.

As so often, a feature not documented doesn't exist for the common user.